javascript/javascript3/week2/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>js3 w2</title>
+</head>
+
+<body>
+    <div class="calc">
+        <h1 class="title">Currency calculator</h1>
+        <div> Amount:</div>
+        <input type="number" placeholder="YOUR VALUE" class="input">
+        <div>Convert from:</div>
+        <select class="options"></select>
+        <div>Convert to:</div>
+        <select class="options2"></select>
+        <button class="button">Convert</button>
+        <div class="output"></div>
+    </div>
+
+    <script src="script.js"></script>
+</body>
+
+</html>
```

Comment:
It would be nice if you format your code, for example with Prettier  https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode.